tests/atomistics/job/test_transform_trajectory.py

```diff
@@ -0,0 +1,76 @@
+import os
+import shutil
+import unittest
+from abc import ABC
+import numpy as np
+from pathlib import Path
+from pyiron_base._tests import TestWithCleanProject
+from pyiron_base import ProjectHDFio
+from pyiron_atomistics.atomistics.job.atomistic import AtomisticGenericJob, Trajectory, TransformTrajectory
+from pyiron_atomistics.atomistics.structure.atoms import Atoms, CrystalStructure
+
+
+class ToyAtomisticJob(AtomisticGenericJob, ABC):
+
+    def _check_if_input_should_be_written(self):
+        return False
+
+    def run_static(self):
+        self.save()
+        self.status.running = True
+        self.status.finished = True
+        self.to_hdf()
```

Comment:
This should not be required. The save should already be called by default! 
```suggestion
    def run_static(self):
        self.status.finished = True
        self.to_hdf()
```
should be enough.